src/app/components/cds/inputs/Select.js

```diff
@@ -26,22 +31,38 @@ const Select = ({
         { required && <span className={inputStyles.required}>*<FormattedMessage id="textfield.required" defaultMessage="Required" description="A label to indicate that a form field must be filled out" /></span>}
       </div>
     )}
-    <div className={`${inputStyles['input-container']} ${styles['select-container']}`}>
-      { iconLeft && (
-        <div className={inputStyles['input-icon-left-icon']}>
-          {iconLeft}
+    <div className={styles['input-wrapper']}>
+      <div className={`${inputStyles['input-container']} ${styles['select-container']}`}>
+        { iconLeft && (
+          <div className={inputStyles['input-icon-left-icon']}>
+            {iconLeft}
+          </div>
+        )}
+        <select
+          id="check-select__input"
+          className={`typography-body1 ${styles.input} ${variant === 'outlined' && styles.outlined} ${error && styles.error} ${iconLeft && styles['input-icon-left']}`}
+          disabled={disabled}
+          {...inputProps}
+        >
+          {children}
+        </select>
+        <div className={inputStyles['input-icon-right-icon']}>
+          <ChevronDownIcon />
         </div>
-      )}
-      <select
-        className={`typography-body1 ${styles.input} ${variant === 'outlined' && styles.outlined} ${error && styles.error} ${iconLeft && styles['input-icon-left']}`}
-        disabled={disabled}
-        {...inputProps}
-      >
-        {children}
-      </select>
-      <div className={inputStyles['input-icon-right-icon']}>
-        <ChevronDownIcon />
       </div>
+      { onRemove ?
+        <Tooltip title={<FormattedMessage id="select.removeSelection" defaultMessage="Undo selection" description="Tooltip for button on Select component to remove current selection" />}>
+          <IconButton
+            className={cx('select__clear-button', styles['select-clear-button'])}
```

Comment:
What does `cx` do?

Reply:
@caiosba this is a shortcut variable name for the new classNames package that we added to make it easier to manage multiple classnames, especially when they are applied with conditional logic. the cx is from the example repo...I think the author meant it as shorthand for classnames bind, since in this example alex is using both a global `classname select__clear-button` and a classname bound to a css modules file `select-clear-button`

you can check out some info here with links to the package:
[https://github.com/meedan/check-web/pull/1468](https://github.com/meedan/check-web/pull/1468)

Reply:
Ah cool, thanks! I'll start using it too.